<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExamView</class>
 <widget class="QWidget" name="ExamView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1054</width>
    <height>672</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#ExamView{
	background-color: white;
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="0">
   <property name="spacing">
    <number>4</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="styleSheet">
      <string notr="true">QSplitter#splitter::handle{
	background-color: rgb(20,196,188);
}
QSplitter#splitter::margin{
	background-color: rgb(20,196,188);
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="handleWidth">
      <number>4</number>
     </property>
     <property name="childrenCollapsible">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="widgetLeft" native="true">
      <property name="maximumSize">
       <size>
        <width>340</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QWidget#widgetLeft{
	background-color: white;
}

QPushButton{
	color: #FFFFFF;
	border: 2px solid rgb(24, 204, 192);
	background-color: rgb(20,196,188);
}
QPushButton:hover{
	background-color: rgb(22,218,208);
}
QPushButton:pressed{
	background-color: rgb(17,171,164);
}</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,0,0,1,0,0,0">
       <item>
        <widget class="QGroupBox" name="boxExamInfo">
         <property name="title">
          <string>考试信息</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="layoutExamName" stretch="0,1">
            <item>
             <widget class="QLabel" name="labelExamNameText">
              <property name="text">
               <string>考试名称:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="labelExamName">
              <property name="text">
               <string/>
              </property>
              <property name="textFormat">
               <enum>Qt::PlainText</enum>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="layoutQuesCnt" stretch="0,1">
            <item>
             <widget class="QLabel" name="labelQuesCntText">
              <property name="text">
               <string>题目数量:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="labelQuesCnt">
              <property name="text">
               <string/>
              </property>
              <property name="textFormat">
               <enum>Qt::PlainText</enum>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="layoutCurTime" stretch="0,1">
            <item>
             <widget class="QLabel" name="labelCurTimeText">
              <property name="text">
               <string>当前时间:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="labelCurTime">
              <property name="text">
               <string/>
              </property>
              <property name="textFormat">
               <enum>Qt::PlainText</enum>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="layoutStartTime" stretch="0,1">
            <item>
             <widget class="QLabel" name="labelStartTimeText">
              <property name="text">
               <string>开始时间:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="labelStartTime">
              <property name="text">
               <string/>
              </property>
              <property name="textFormat">
               <enum>Qt::PlainText</enum>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="layoutEndTime" stretch="0,1">
            <item>
             <widget class="QLabel" name="labelEndTimeText">
              <property name="text">
               <string>结束时间:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="labelEndTime">
              <property name="text">
               <string/>
              </property>
              <property name="textFormat">
               <enum>Qt::PlainText</enum>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="layoutSrvStat" stretch="0,1">
            <item>
             <widget class="QLabel" name="labelSrvStatText">
              <property name="text">
               <string>连接状态:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="labelSrvStat">
              <property name="text">
               <string/>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="boxStuInfo">
         <property name="title">
          <string>考生信息</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="layoutStuName" stretch="0,1">
            <item>
             <widget class="QLabel" name="labelStuNameText">
              <property name="text">
               <string>考生名称:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="labelStuName">
              <property name="text">
               <string/>
              </property>
              <property name="textFormat">
               <enum>Qt::PlainText</enum>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="layoutUploadTime" stretch="0,1">
            <item>
             <widget class="QLabel" name="labelUploadTimeText">
              <property name="text">
               <string>最后一次上传:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="labelUploadTime">
              <property name="text">
               <string>暂无</string>
              </property>
              <property name="textFormat">
               <enum>Qt::PlainText</enum>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnStart">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>开始考试</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widgetButtons" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="btnCheck">
            <property name="text">
             <string>检查漏题</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnUpload">
            <property name="text">
             <string>上传作答</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verSpacerBtn">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>16</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="btnFinish">
            <property name="text">
             <string>交卷并结束</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verSpacerLeft">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>387</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>1</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btnReconnect">
         <property name="text">
          <string>重新连接</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnExit">
         <property name="text">
          <string>登出</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="widgetRight" native="true">
      <property name="styleSheet">
       <string notr="true">QWidget#widgetRight{
	background-color: white;
}</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QScrollArea" name="widgetExam">
         <property name="styleSheet">
          <string notr="true">QScrollArea{
	border: none;
}</string>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="widgetExamContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>708</width>
            <height>670</height>
           </rect>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
